<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_RPEvent</class>
 <widget class="QDialog" name="Dialog_RPEvent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>346</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Event definition</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Modify:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="2" colspan="2">
       <widget class="QComboBox" name="combo_powerProfileChange">
        <item>
         <property name="text">
          <string>No change</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QComboBox" name="combo_powerLevelChange">
        <item>
         <property name="text">
          <string>No change</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QComboBox" name="combo_fanChange">
        <item>
         <property name="text">
          <string>No change</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="spin_fixedFanSpeed">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="l_fan">
        <property name="text">
         <string>Set fan:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="l_powerLevel">
        <property name="text">
         <string>Set power level:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="l_profile">
        <property name="text">
         <string>Set power profile to:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="edt_eventName"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Event name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="cb_enabled">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Enabled</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="combo_eventTrigger">
       <item>
        <property name="text">
         <string>Temperature</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Binary</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Event trigger</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
       <widget class="QComboBox" name="combo_sensorInstance">
       </widget>
     </item>
     <item row="3" column="0">
       <widget class="QLabel" name="label_sensorInstance">
           <property name="text">
             <string>Temperature Sensor</string>
           </property>
       </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Activate above this temperature:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_tempActivate">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Binary (executing this binary will trigger event):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="edt_binary"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_setBinary">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btn_save">
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="btn_cancel">
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>combo_eventTrigger</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
